### PR TITLE
Translate contacts birthday

### LINKED ISSUE #23982: [Tanslation] [CalDAV] Translate the Name of the Birthday Calendar
Since 9.0.1 we call contacts_birthdays "Contacts birthday", but i think we should also translate this :wink:

### PR DESCRIPTION
**Pull request by DeepDiver1975**
fixes #23982

### CODE REVIEW

File: apps/dav/lib/caldav/calendarhome.php [hunk 1]
```diff
@@ -31,34 +32,42 @@
 
 class CalendarHome extends \Sabre\CalDAV\CalendarHome {
 
+	/** @var \OC_L10N */
```
Comment (nickvergessen) [inline apps/dav/lib/caldav/calendarhome.php]: OCP\IL10N

File: apps/dav/lib/caldav/calendar.php [hunk 2]
```diff
@@ -22,17 +22,19 @@
...
 
 class Calendar extends \Sabre\CalDAV\Calendar implements IShareable {
 
-	public function __construct(BackendInterface $caldavBackend, $calendarInfo) {
+	public function __construct(BackendInterface $caldavBackend, $calendarInfo, IL10N $l10n) {
 		parent::__construct($caldavBackend, $calendarInfo);
 
 		if ($this->getName() === BirthdayService::BIRTHDAY_CALENDAR_URI) {
 			$this->calendarInfo['{http://sabredav.org/ns}read-only'] = true;
+			$this->calendarInfo['displayname'] = $l10n->t('Contact birthdays');
```
Comment (nickvergessen) [inline apps/dav/lib/caldav/calendar.php]: You mean `{DAV:}displayname` ?
Otherwise it's not working for me

**Comment by DeepDiver1975** [comment]
@nickvergessen addressed - THX

**Comment by nickvergessen** [comment]
:+1:

**Comment by aspdye** [comment]
Backport? :grin:

**Comment by DeepDiver1975** [comment]
We should only backport critical fixes

**Comment by aspdye** [comment]
Okay :wink:

**Pull request merged**